commands/utils/utils.go

```diff
@@ -168,3 +169,26 @@ func GetCompatibleOutputWriter(provider vcsutils.VcsProvider) OutputWriter {
 	}
 	return &StandardOutput{}
 }
+
+// RemoveDowngradedVersions removes fix suggestions which are downgraded versions of current
+// trimming brackets as some xray api returns with or without brackets it breaks the compare function
+func RemoveDowngradedVersions(vul *formats.VulnerabilityOrViolationRow) {
+	if len(vul.FixedVersions) == 0 {
+		return
+	}
+	upgradeVersions := make([]string, 0)
+	effectedVersion := version.NewVersion(removeBrackets(vul.ImpactedDependencyVersion))
+	for _, suggestedFixVersion := range vul.FixedVersions {
+		suggestedFixVersionTrimmed := removeBrackets(suggestedFixVersion)
+		if !effectedVersion.AtLeast(suggestedFixVersionTrimmed) {
+			upgradeVersions = append(upgradeVersions, suggestedFixVersion)
+		}
+	}
+	vul.FixedVersions = upgradeVersions
+}
+
+func removeBrackets(suggestedFixVersion string) string {
+	replacer := strings.NewReplacer("[", "", "]", "") //trims brackets if exists
```

Comment:
The comment in this line isn't according to our guidelines:
1. Should be above the line it describes.
2. Should have a white-space after //
3. Should start with an upper case letter